### DISCUSSION TITLE
Cannot see static index.html pointing to directory (e.g. `http://localhost/docs/`)

### DISCUSSION THREAD

**Discussion by ManuDoni** [php/frankenphp#620] [Q&A] [answered]
Hi, I'm working on an API that is documented using https://github.com/knuckleswtf/scribe.
Scribe puts static html, js and css files inside the `public/docs` folder.
If I go directly to http://localhost/docs/index.html I can see the documentation, but if I go to http://localhost/docs/ the server gives me a 404 (the laravel one).
Is there a way to tell Caddy that directories should use index files? Or there are other solutions?
Thank you in advance

**Comment by dunglas** [comment]
You can use the `index` option of the `php_server` directive to change the default index.

**Reply by ManuDoni** [reply]
Where can i set that option if I'm using octane? Sorry I don't know much about caddy

**Reply by ManuDoni** [reply]
I just saw that i can tell octane to use another caddyfile using the `--caddyfile` option.
Thank you!

**Reply by ManuDoni** [reply, marked as answer]
This is my solution:
I copied the Caddyfile stub and modified the `php_server` directive this way:
```
php_server {
    index frankenphp-worker.php
    # Required for the public/storage/ directory...
    resolve_root_symlink
    # Required to serve index.html using only the folder
    try_files {path} {path}/index.html {path}/index.php frankenphp-worker.php
}
```
I only added the `try_files {path} {path}/index.html {path}/index.php frankenphp-worker.php`

Then i set my Caddyfile using the `--caddyfile` option of the `php artisan octane:start` command

**Reply by ManuDoni** [reply]
@dunglas sorry if I bother you, but maybe we should add the `index.html` to `try_files` in the default php_server configuration?

**Reply by dunglas** [reply]
Indeed, good idea. Would you like to contribute a patch? We should also add `index.txt` (Caddy defaults are `index.html` and `index.txt`).

**Reply by ManuDoni** [reply]
Yes, I'm not a golang developer but i'll try
